pom.xml

```diff
@@ -124,7 +124,7 @@
              </plugin>
              <plugin>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.4.1</version>
+                <version>2.5.4</version>
```

Comment:
From my testing, this change here is the likely cause of the test failures on Travis. Since this change is not necessary to the bug fix, let's leave the maven-assembly-plugin version number at 2.4.1, and leave the wondering what went wrong for another day?